Makefile.dep

```diff
@@ -68,7 +68,11 @@ ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
   USEMODULE += ble_6lowpan
   USEMODULE += gnrc_sixlowpan
   USEMODULE += gnrc_sixlowpan_iphc
+  USEMODULE += gnrc_ipv6_nib_6ln
   USEMODULE += gnrc_ipv6_default
+  # prevent application from being a router
+  # TODO: maybe find a nicer solution in future build system update
```

Comment:
Improvements on this message, would not block for this. Its just to show more of the magic to users.

Could it be `assert should not be a router` and leave it to the user configuration to make sure its not a router ?
I do not know what triggers this inclusion.

Or just, if its required to remove routing modules, print a warning to say "routing disabled" for BLE.

Reply:
@cladmi 
I tried to add the following
```
  # prevent application from being a router
  # TODO: maybe find a nicer solution in future build system update
  $(info  WARNING: Routing is disabled when softdevice is used)
  USEMODULE := $(filter-out gnrc_ipv6_router% gnrc_rpl netstats_rpl auto_init_gnrc_rpl,$(USEMODULE))
```
in Makefile.dep
But the message is output twice

```
dylan@dylan-K53SD ~/Software/RIOT/examples/gnrc_networking $ make BOARD=nrf52dk
WARNING: Routing is disabled when softdevice is used
WARNING: Routing is disabled when softdevice is used
Building application "gnrc_networking" for "nrf52dk" with MCU "nrf52"
```

Reply:
try ```$(warning ...)``` instead of info, that should make the output go to stderr.